doc/source/whatsnew/v0.25.0.rst

```diff
@@ -28,6 +28,24 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Pandas 0.25.0 includes a number of API breaking changes.
+
+.. _whatsnew_0250.api_breaking.deps:
+
+Increased minimum versions for dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We have updated our minimum supported versions of dependencies (:issue:`25036`).
+If installed, we now require:
+
++-----------------+-----------------+----------+
+| Package         | Minimum Version | Required |
++=================+=================+==========+
+| IPython         | 3.0.0           |          |
```

Comment:
is there a reason we don't just go to the py3 only version? (6.x I think)?

Reply:
No reason. This section can likely be remove entirely.